layouts/partials/download/content-linux.html

```diff
@@ -11,7 +11,7 @@ <h3>Get O3DE for Linux (Preview)</h3>
         <h3><i class="fab fa-github"></i>Customize your Engine</h3>
         <p>Looking to dive deep into O3DE? Get the engine sources, build O3DE, and start working with the code now.</p>
         <p id="github-page-button-windows">
-          <a href="https://github.com/o3de/o3de/main" class="btn-xl-block btn-lg btn-green text-white text-center" target="_blank">Fork from GitHub</a>
+          <a href="https://github.com/o3de/o3de/tree/main" class="btn-xl-block btn-lg btn-green text-white text-center" target="_blank">Fork from GitHub</a>
```

Comment:
Consider using https://github.com/o3de/o3de/ instead of pointing the to a specific branch. I left it in there because that is what the intent behind the original link was.